assets/css/general.css

```diff
@@ -290,6 +290,12 @@ footer {
   font-size: 2rem;
 }
 
+.sponsors-link {
```

Comment:
Maybe use `h3 > a` instead of a class? Presumably any time we have a link in an H3, we want this styling.

Also should we also have `text-decoration-line: none;` ?

Reply:
The only reason I didn't use the "h3 > a" was because I don't have knowledge on the website, and I don't want u to create some other h3 with a link and randomly get this style..

As per line decoration I was told to make it as the design of the sponsors in the nav is, every link had line decoration so I made it as they have it.

Reply:
> The only reason I didn't use the "h3 > a" was because I don't have knowledge on the website, and I don't want u to create some other h3 with a link and randomly get this style..

Fair point. AFAIK we don't have other H3 headings with links in. We do have H2 headings with links, in the ["Blog"](https://openwebdocs.org/content/posts/) page:

<img width="1431" alt="Screen Shot 2022-09-09 at 10 36 10 AM" src="https://user-images.githubusercontent.com/432915/189411226-0169abe3-916e-47c8-a436-5dc0970b0d33.png">

If we extended this style to other headings, like:

```css
:is(h1, h2, h3, h4, h5, h6) > a {
  color: var(--foreground);
}

:is(h1, h2, h3, h4, h5, h6) > a:hover {
  color: var(--link);
}
```

...then that page is like:

<img width="1442" alt="Screen Shot 2022-09-09 at 10 37 30 AM" src="https://user-images.githubusercontent.com/432915/189411477-af3aa5d3-58c0-4637-8a18-701c5434f093.png">

...which I think is fine, but idk.

> As per line decoration I was told to make it as the design of the sponsors in the nav is, every link had line decoration so I made it as they have it.

I had read "Please keep the styling/look it already has as a header" as meaning, among other things, don't underline it like a link. But maybe @robnyman can clarify :).

Reply:
I'm happy to do any changes that are asked on the issue that I worked on, as per my personal opinion i think those h2 headings should stay the purple color as it gives better contrast to the content. :)

Reply:
@wbamberg @abitsalihu I don't have a really strong opinion on whether there should be an underline visible at all times, or not. I think it looks better without, but the benefit for sponsors might be that it's more of a recognizable link. So, I'm ok with either. :-)

Reply:
@wbamberg @abitsalihu I don't have a really strong opinion on whether there should be an underline visible at all times, or not. I think it looks better without, but the benefit for sponsors might be that it's more of a recognizable link. So, I'm ok with either. :-)

Reply:
I'm ok with an underline, for now. :-)
Then we can change it later if people feel strongly about it. Thanks!